queries/futures/useGetFuturesDailyTrades.ts

```diff
@@ -1,31 +1,28 @@
-import { utils as ethersUtils } from 'ethers';
 import { useQuery, UseQueryOptions } from 'react-query';
 import { useRecoilValue } from 'recoil';
 
 import QUERY_KEYS from 'constants/queryKeys';
 import { appReadyState } from 'store/app';
+import { marketInfoState } from 'store/futures';
 import { isL2State, networkState } from 'store/wallet';
 import { calculateTimestampForPeriod } from 'utils/formatters/date';
-import { FuturesMarketAsset } from 'utils/futures';
 import logError from 'utils/logError';
 
 import { DAY_PERIOD } from './constants';
 import { getFuturesTrades } from './subgraph';
 import { getFuturesEndpoint } from './utils';
 
-const useGetFuturesDailyTradeStatsForMarket = (
-	marketAsset: FuturesMarketAsset | null,
```

Comment:
I feel like it makes sense for this hook to take a market as a param so it can then be re-used in any place, otherwise it's restricted to only being used for the single market in recoil.

Reply:
good point, I added it back. I was trying to avoid passing recoil values as params for the hooks we are still calling from components. This one is only used in one place (`MarketDetails`) so it wouldn't make a difference unless we reuse this hook elsewhere.

Reply:
Right now we're able to pull all or most contract data for all markets in a single hook and store in a recoil object. I'm thinking we could use a similar pattern for subgraph data so we don't call the hooks all over the place in components.